debian/control

```diff
@@ -1,9 +1,9 @@
 Package: dokku
-Version: 0.3.12
+Version: 0.4.3
```

Comment:
Huh, so it does. There's a part of me that would like to move a bit more to standard .deb build tools rather than the ad-hoc stuff in deb.mk, but I'm still learning those bits myself :)